Dockerfile

```diff
@@ -1,23 +1,24 @@
-# Use Ubuntu 18.04 (will be supported until April 2023)
-FROM ubuntu:bionic
+# Use Ubuntu 22.04 (will be supported until April 2027)
+FROM ubuntu:jammy
 
 # Add openMVG binaries to path
 ENV PATH $PATH:/opt/openMVG_Build/install/bin
 
 # Get dependencies
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y tzdata; \
+  apt-get install -y \
   cmake \
   build-essential \
   graphviz \
   git \
   coinor-libclp-dev \
   libceres-dev \
   libflann-dev \
-  liblemon-dev \
```

Comment:
Why removing this package?
- It still exists `liblemon-dev` https://packages.ubuntu.com/search?suite=jammy&searchon=names&keywords=lemon

Reply:
I will merge your PR but add back this module